src/index.html

```diff
@@ -42,9 +42,6 @@ <h1 class="govuk-heading-xl">
   {% if current_alerts %}
   <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
   {{ banner(current_alerts | length, data_last_updated) }}
-  {% else %}
-    <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
-    {{ planned_tests_banner(1, "Tuesday 25 May") }}
```

Comment:
Good spot Ben, this is good that we've removed it as after testing this link should get removed.

@quis and @karlchillmaid not that important but could be worth thinking what should be shown on the planned tests page (even though we won't link to it directly) during and after the current alert

Reply:
PR prepared for after East Suffolk here: https://github.com/alphagov/notifications-govuk-alerts/pull/74

Reply:
I've dropped the commit to remove the else clause. If we deploy #74 between 1-2PM, then it'll show up automatically when the alert expires.